hack-agenda.md

```diff
@@ -0,0 +1,14 @@
+#Hack-a-thon Agenda
+
+##Thursday
+- What is victory, clarify the vision and the first milestone targets
+- API design generally, detailed API design for foundational components
+- Animation Model!
+- Take stock of what we already have / can reuse
+- Hack
+
+##Friday
+- Talk to Paula about base styling in Radium
```

Comment:
We need a victory component styleguide, that's the highest guide anyway, and it should be a source of data viz best practices. Ie., if we have hexbins they should use standard victory colors, which should be colorblind friendly and good contrast and muted so that you can overlay tooltips etc etc.